docs/en_US/Tuner/BuiltinTuner.rst

```diff
@@ -58,98 +58,56 @@ Note: Please follow the format when you write your ``config.yml`` file. Some bui
-
-:raw-html:`<br>`
+TPE, as a black-box optimization, can be used in various scenarios and shows good performance in general. Especially when you have limited computation resources and can only try a small number of trials. From a large amount of experiments, we found that TPE is far better than Random Search. `Detailed Description <./TpeTuner.rst>`__
 
 :raw-html:`<a name="Random"></a>`
 
 Random Search
 ^^^^^^^^^^^^^
 
-..
-
-   Built-in Tuner Name: **Random**
-
-Random search is suggested when each trial does not take very long (e.g., each trial can be completed very quickly, or early stopped by the assessor), and you have enough computational resources. It's also useful if you want to uniformly explore the search space. Random Search can be considered a baseline search algorithm. `Detailed Description <./HyperoptTuner.rst>`__
-
-:raw-html:`<br>`
+Random search is suggested when each trial does not take very long (e.g., each trial can be completed very quickly, or early stopped by the assessor), and you have enough computational resources. It's also useful if you want to uniformly explore the search space. Random Search can be considered a baseline search algorithm. `Detailed Description <./RandomTuner.rst>`__
 
 :raw-html:`<a name="Anneal"></a>`
 
 Anneal
 ^^^^^^
 
-..
-
-   Built-in Tuner Name: **Anneal**
-
 Anneal is suggested when each trial does not take very long and you have enough computation resources (very similar to Random Search). It's also useful when the variables in the search space can be sample from some prior distribution. `Detailed Description <./HyperoptTuner.rst>`__
 
-:raw-html:`<br>`
-
 :raw-html:`<a name="Evolution"></a>`
 
 Naïve Evolution
 ^^^^^^^^^^^^^^^
 
-..
-
-   Built-in Tuner Name: **Evolution**
-
 Its computational resource requirements are relatively high. Specifically, it requires a large initial population to avoid falling into a local optimum. If your trial is short or leverages assessor, this tuner is a good choice. It is also suggested when your trial code supports weight transfer; that is, the trial could inherit the converged weights from its parent(s). This can greatly speed up the training process. `Detailed Description <./EvolutionTuner.rst>`__
 
-:raw-html:`<br>`
-
 :raw-html:`<a name="SMAC"></a>`
 
 SMAC
 ^^^^
 
-..
-
-   Built-in Tuner Name: **SMAC**
-
-
 **Please note that SMAC doesn't support running on Windows currently**. For the specific reason, please refer to this `GitHub issue <https://github.com/automl/SMAC3/issues/483>`__.
 
 Similar to TPE, SMAC is also a black-box tuner that can be tried in various scenarios and is suggested when computational resources are limited. It is optimized for discrete hyperparameters, thus, it's suggested when most of your hyperparameters are discrete. `Detailed Description <./SmacTuner.rst>`__
 
-:raw-html:`<br>`
-
 :raw-html:`<a name="Batch"></a>`
 
 Batch Tuner
 ^^^^^^^^^^^
 
-..
-
-   Built-in Tuner Name: BatchTuner
-
-
 **Suggested scenario**
```

Comment:
do we need this line?

Reply:
I don't know. It's not a style issue and has not been discussed.